220-ofcom-22000.md

```diff
@@ -1,4 +1,58 @@
 # 2.20. OFCOM \(-22000\)
+## ofcom\_odbc\: SQLConnect failed
 
+* Description
+  * This error occurs when ODBC is failed to run.
+* Action 
+  * Check if tibero is running and boot if it is not.
+  
+  ---
+  
+  ### Case 1: ODBC failed
+  
+  * Problem 
+    * dataset migration fails with below message
+    ```
+    ofcom_odbc: SQLConnect failed
+    - State: ERRCL
+    - Native Error: -2131
+    - Error Message: [unixODBC] Generic I/O error.
+    012504 E MIG0201E ams_initialize() failed. - rc = -22110
+    ```
+  * Solution
+    1. Check if tibero is running, by using ps -eLf | grep tibero 
+    ```bash
+    jh_lee@jh_lee:~$ ps -eLf | grep tibero
+    jh_lee    2637  2334  2637  0    1 02:56 pts/0    00:00:00 grep --color=auto tibero
+    ```
```

Comment:
Change the statement 
If you see just one line, it means it is not running. -> If you see no tbsvr process running, it means the tibero has not been booted up.